.github/workflows/performance.yml

```diff
@@ -178,74 +199,85 @@ jobs:
       - name: Run performance tests (current commit)
         run: npm run test:performance
 
-      - name: Print performance tests results
-        run: node ./tests/performance/results.js
-
-      - name: Check out target commit (target branch or previous commit)
+      - name: Get previous build (target branch or previous commit)
+        id: get-previous-build
         run: |
-          if [[ -z "$TARGET_REF" ]]; then
-            git fetch -n origin $TARGET_SHA
-          else
-            git fetch -n origin $TARGET_REF
-          fi
-          git reset --hard $TARGET_SHA
+          echo "artifact_id=$(gh api repos/WordPress/wordpress-develop/actions/artifacts?name="wordpress-build-$TARGET_SHA" --jq ".artifacts[] .id")" >> $GITHUB_OUTPUT
```

Comment:
Would we be able to utilize actions/github-acripts here? It handles a few things for us, such as logging,  supporting automatic retry when a request fails and handling the token automatically. On first glance, it looks like the `listArtifactsForRepo()` [endpoint supports supplying a name](https://octokit.github.io/rest.js/v20#actions-list-artifacts-for-repo).